routers/intake.py

```
@@ -58,6 +59,42 @@ async def intake_message(
+                                request_type=request_type,
+                                parsed_input=parsed,
+                                options=options,
+                                info={}
+                            )
+                            request_store.save(stored)
+                            await status_message.edit_text(
+                                text.FORMAT_SELECTION,
+                                reply_markup=format_keyboard(token, options),
+                            )
+                            return
+            except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)